ci/build_container/build_container.sh

```diff
@@ -46,3 +46,10 @@ export THIRDPARTY_BUILD=/thirdparty_build
 export CPPFLAGS="-DNDEBUG"
 echo "Building opt deps ${DEPS}"
 "$(dirname "$0")"/build_and_install_deps.sh ${DEPS}
+
+echo "Building Bazel-managed deps (//bazel/external:all_external)"
+BAZEL_OPTIONS="--output_user_root=/bazel-prebuilt/BAZEL_ROOT --output_base=/bazel-prebuilt/BAZEL_OUTPUT"
+cd /bazel-prebuilt
+for BAZEL_MODE in opt dbg fastbuild; do
```

Comment:
Have you looked into the size of the Docker image? We're now doing 3x the build of external deps. We briefly went from 1 -> 2 and that created almost 50% growth. Please provide stats in the review and I think @mattklein123 might want to review this for sanity.

Reply:
On `master` the image is 2.34GB, on this branch the image is `2.17GB`. Bazel is fairly good at avoiding copies of files that aren't different between compilation modes.

I'm also preparing another PR to trim the image by reducing Debian deps with `apt-get --no-install-recommends`, and have filed a Bazel bug to allow use of the headless JDK because we don't need X11 (https://github.com/bazelbuild/bazel/issues/3761).

Reply:
Thanks, reducing image size is a nice improvement.